### PR TITLE
merge pluton

### PR DESCRIPTION
**Pull request by pbx0**
cc @marineam @ethernetdan

### CODE REVIEW

File: cmd/pluton/pluton.go [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+
+	"github.com/coreos/pkg/capnslog"
+	"github.com/spf13/cobra"
+)
+
+var (
+	plog = capnslog.NewPackageLogger("github.com/coreos-inc/pluton", "kola")
+
+	root = &cobra.Command{
+		Use:   "kola [command]",
```
Comment (marineam) [inline cmd/pluton/pluton.go]: pluton

File: cmd/pluton/pluton.go [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+		Use:    "run [glob pattern]",
+		Short:  "Run run kola tests by category",
+		Long:   "run all kola tests (default) or related groups",
+		Run:    runRun,
+		PreRun: preRun,
+	}
+
+	cmdList = &cobra.Command{
+		Use:   "list",
+		Short: "List kola test names",
```
Comment (marineam) [inline cmd/pluton/pluton.go]: pluton

File: cmd/pluton/pluton.go [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+	err := syncOptions()
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Error: %v\n", err)
+		os.Exit(3)
+	}
+}
+
+func runRun(cmd *cobra.Command, args []string) {
+	if len(args) > 1 {
+		fmt.Fprintf(os.Stderr, "Extra arguements specified. Usage: 'kola run [glob pattern]'\n")
```
Comment (marineam) [inline cmd/pluton/pluton.go]: pluton

File: pluton/spawn/bootkube.go [hunk 4]
```diff
@@ -0,0 +1,295 @@
...
+
+	"github.com/coreos/mantle/kola/cluster"
+	"github.com/coreos/mantle/kola/tests/etcd"
+	"github.com/coreos/mantle/platform"
+	"github.com/coreos/mantle/pluton"
+
+	"github.com/coreos/pkg/capnslog"
+)
+
+var plog = capnslog.NewPackageLogger("github.com/coreos-inc/pluton", "spawn")
```
Comment (marineam) [inline pluton/spawn/bootkube.go]: coreos/mantle, pluton/spawn

File: pluton/tests/bootkube/registry.go [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+
+package bootkube
+
+import (
+	"github.com/coreos/pkg/capnslog"
+
+	"github.com/coreos/mantle/kola/register"
+)
+
+var plog = capnslog.NewPackageLogger("github.com/coreos-inc/pluton", "tests/bootkube")
```
Comment (marineam) [inline pluton/tests/bootkube/registry.go]: coreos/mantle, pluton/tests

**Comment by pbx0** [comment]
PTAL -- I think I fixed up the last of the names.

**Pull request merged**